private-undistributed.yml

```diff
@@ -2,129 +2,95 @@
 # This is to circumvent GitHub having a flaky license discovery for some libraries.
 # A script is present in the scripts folder to generate the purl for a specific groupId.
 allow-dependencies-licenses:
-  # All Spring Framework, Spring Boot and Spring Cloud
```

Comment:
Sorry about the big diff, but basically the packages from `org.springframework.data` were allowed to the allow-list and all the informations here are now sorted alphabetically.